HEN_HOUSE/scripts/configure

```diff
@@ -2026,6 +2026,14 @@ str_header=$(printf "$str_header" | while read line; do
     printf "\"\n"
 done)
 
+GIT_HASH=" "
+GIT_BRANCH=" "
+CONFIG_TIME=`date -u +'%Y-%m-%d %H:%M:%S UTC'`
+if type "git" > /dev/null 2>&1; then
+    GIT_HASH=`git rev-parse --short=7 HEAD`
```

Comment:
Perhaps add a conditional `if git rev-parse --is-inside-work-tree >/dev/null 2>&1; then`, because users might have git on their system yet will not be using the git clone for EGSnrc.